README.md

```diff
@@ -92,13 +92,21 @@ databases.
     post: "..."
 ```
 
-Binary columns can be represented as hexadecimal strings (should start with `0x`):
+Binary columns can be represented as hexadecimal strings (should start with `0x` and will be automatically converted to `[]byte`):
 
 ```yaml
 - id: 1
   binary_column: 0x1234567890abcdef
 ```
 
+String values matching date/time formats (e.g., "2025-08-15", "20250815", "15/08/2025") will be automatically converted to `time.Time`. Use `RAW=` prefix to prevent conversion:
```

Comment:
could you link https://github.com/go-testfixtures/testfixtures/blob/master/time.go#L8C1-L27C2 here? It may be beneficial to check what formats are possible